deepseek-r1, gemma3, gemma4, nemotron3-nano, qwen-vl: add page (#22582)
tldr-pages/tldr: pages/linux/inference-snaps.md

> # inference-snaps

Reviewer: ```suggestion
# Inference snaps
```
This could be natural language as it isn't describing a direct command.

> > Inference Snaps allow chatting with generative AI Language Models that runs locally without internet.

Reviewer: ```suggestion
> Chat with generative AI Language Models that runs locally without internet.
```
This removes the redundant text and conforms it with the style guide.

> > Here inference-snap is one of the supported models: deepseek-r1|gemma3|gemma4|nemotron3-nano|qwen-vl .

Reviewer: ```suggestion
> Here inference-snap is one of the supported models: `deepseek-r1`, `gemma3`, `gemma4`, `nemotron3-nano`, `qwen-vl` .
```
